trulens_eval/requirements.txt

```diff
@@ -3,6 +3,7 @@ python-dotenv
 langchain
 sqlalchemy
 alembic
+overrides
```

Comment:
nit: sorry about pointing you to the @override annotation. I think in python you can just not annotate it at all and it'll work so this probably is not needed